apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2023-05-15T23:18:19Z"
-  build_hash: 8f3ba427974fd6e769926778d54834eaee3b81a3
+  build_date: "2023-05-09T20:14:55Z"
+  build_hash: 811e30bb8efe2855f79e6e946039247e23c0d03b
   go_version: go1.19
-  version: v0.26.1
-api_directory_checksum: a9fcef68210dd72b4b2e37052f2c1a9e971326c6
+  version: v0.25.0-9-g811e30b-dirty
```

Comment:
Can you regenerate this using `code-generator@v0.26.1`

Reply:
Also we might want to bring your changes to code-generator before merging this

Reply:
Opened [PR for code-generator](https://github.com/aws-controllers-k8s/code-generator/pull/451)